src/loaders/compute_tools/busco/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM continuumio/miniconda3:24.5.0-0
+
+# NOTE: If the tool version changes, ensure the metadata information saved after running the tool in the _run_busco_single method is updated
+ARG BUSCO_VER=5.7.1
+ENV CONDA_ENV busco-$BUSCO_VER
+
+# Add Bioconda and Conda-Forge channels
+RUN conda config --add channels bioconda
+RUN conda config --add channels conda-forge
+
+# Install BUSCO
+# Certain dependencies (e.g., dendropy, sepp) are only compatible with Python versions up to 3.9.
+ARG PYTHON_VER=3.9
+RUN conda create -n $CONDA_ENV python=$PYTHON_VER
+RUN conda install -n $CONDA_ENV pandas=2.2.2 jsonlines=4.0.0 mamba=1.5.8 pyyaml=6.0.1
+RUN conda run -n $CONDA_ENV mamba install -c bioconda -c conda-forge -y busco=$BUSCO_VER
```

Comment:
I'm not familiar enough with conda / mamba to understand exactly what's going on here, can you explain?

Reply:
added comments.

Reply:
Why the `conda run`? The linked instructions have mamba executed separately

Reply:
to ensure mamba is called from the correct conda environment.